src/common/providers/https.ts

```diff
@@ -638,7 +638,7 @@ export async function checkAuthToken(
   if (!authorization) {
     return 'MISSING';
   }
-  const match = authorization.match(/^Bearer (.*)$/);
+  const match = authorization.match(/^Bearer (.*)$/i);
   if (match) {
```

Comment:
nit: maybe instead of indenting the whole function in the general case of "match" we can instead early exit with "invald" if `!match` and simplify the whole function.